### PR TITLE
Keyboard shortcuts for next and previous job

### PR DESCRIPTION
**Pull request by rbillings**
Keyboard shortcuts for issue #57 , beginning with selections for next and previous job.

### CODE REVIEW

**Comment by rbillings** [comment]
Big thanks to @MichalTHEDUDE for https://github.com/mozilla/treeherder-tests/pull/97 which was used as the base of the PR.

**Comment by rbillings** [comment]
Ready for review

File: tests/test_keyboard_shortcuts.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    num_of_jobs = len(all_jobs) - 1
+    random_index = random.randint(0, num_of_jobs)
+
+    # Select random job and job next to it
+    all_jobs[random_index].click()
+    assumed_job_keyword = page.job_details.job_keyword_name
+
+    page.select_next_job()
+
+    assert page.job_details.job_keyword_name == assumed_job_keyword
```
Comment (davehunt) [inline tests/test_keyboard_shortcuts.py]: I think a more suitable check would be to take two consecutive items from a random index of `page.all_jobs[:-1]`, add a `selected` property to the `Job` class, and assert the initial job is selected before the keyboard shortcut is invoked, and the following job is selected after. I testing this locally with the following patch:

```patch
diff --git a/pages/treeherder.py b/pages/treeherder.py
index 3cba24a..e5612ee 100644
--- a/pages/treeherder.py
+++ b/pages/treeherder.py
@@ -290,6 +290,10 @@ class TreeherderPage(Base):
                 self.root.click()
                 self.wait.until(lambda _: self.page.job_details.job_result_status)
 
+            @property
+            def selected(self):
+                return 'selected-job' in self.root.get_attribute('class')
+
     class JobDetails(Region):
 
         _job_details_panel_locator = (By.ID, 'job-details-panel')
diff --git a/tests/test_keyboard_shortcuts.py b/tests/test_keyboard_shortcuts.py
index 125007a..3a11c5c 100644
--- a/tests/test_keyboard_shortcuts.py
+++ b/tests/test_keyboard_shortcuts.py
@@ -13,19 +13,19 @@ def test_next_job_shortcut(base_url, selenium):
     using Right Arrow shortcut, verify job keywords match."""
 
     page = TreeherderPage(selenium, base_url).open()
-    all_jobs = page.all_jobs
+    all_jobs = page.all_jobs[:-1]
 
-    # Check number of jobs
-    num_of_jobs = len(all_jobs) - 1
-    random_index = random.randint(0, num_of_jobs)
+    random_index = random.randint(0, len(all_jobs))
+    jobs = all_jobs[random_index:(random_index + 2)]
 
     # Select random job and job next to it
-    all_jobs[random_index].click()
-    assumed_job_keyword = page.job_details.job_keyword_name
+    jobs[0].click()
+    assert jobs[0].selected
+    assert not jobs[1].selected
 
     page.select_next_job()
-
-    assert page.job_details.job_keyword_name == assumed_job_keyword
+    assert jobs[1].selected
+    assert not jobs[0].selected
```

Comment (rbillings) [inline tests/test_keyboard_shortcuts.py]: This is very helpful, thanks!

**Comment by rbillings** [comment]
Let me know if I should change the title, or move the new tests into a new branch.

**Comment by davehunt** [comment]
@rbillings I'd prefer these new tests to be a separate pull request, as this was reviewed and ready for merge.

**Pull request merged**